pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -248,6 +257,20 @@ def restore_lr_schedulers(self) -> None:
     # ----------------------------------
     # PRIVATE OPS
     # ----------------------------------
```

Comment:
The function is under `private ops` yet we are deprecating it.

Are there any places where we recommend overriding it? Or any docs or public example which uses it?

Reply:
This comment was already there. If I don't deprecate someone is going to comment about it because it was a public method. It does not hurt